README.md

```diff
@@ -1,70 +1,51 @@
-
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
-
-### `npm test`
+# Math Magician Aplication
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+> A single page application with a calculator
 
-### `npm run build`
+## Built With
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
-
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
-
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
-
-### `npm run eject`
-
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
-
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
-
-Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+- HTML5, CSS3, JS ES6
+- React
+- Webpack
+- Linters: ESLint, Stylelint, Webhint
+- GitFlow
+- Jest
 
-You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+## Author
 
-## Learn More
+👤 **Artur Gouveia**
 
-You can learn more in the [Create React App documentation](https://facebook.github.io/create-react-app/docs/getting-started).
+- GitHub: @Arturgouveia1970
+- Twitter: @Arturgouveia10
+- LinkedIn: artur-gouveia-323868197
+   
+## Getting Started
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+To get a local copy up and running follow these simple example steps.
 
-### Code Splitting
+### Prerequisites
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+You must have a browser on your device, a visual code, a git version installed, node.js installed in your device.
 
-### Analyzing the Bundle Size
+### Usage
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+Set up the project by cloning and changing into the repo directory:
+https://github.com/Arturgouveia1970/math-magician-3.git
 
```

Comment:
- [ ] [optional] It would be nice adding instruction on how to run and test the project locally.

Reply:
Hi @danmainah, I will do it. thank you